modules/ROOT/pages/tools/neo4j-admin/upload-to-aura.adoc

```diff
@@ -84,7 +84,7 @@ The `neo4j-admin database upload` command has the following options:
 |
 
 |--from-path=<path>
-|_/path/to/directory-containing-dump_ Path to a directory containing a database dump to upload.
+|_/path/to/directory-containing-dump-or-backup_ Path to a directory containing a database dump or backup file to upload.
```

Comment:
These parts are tested, so we cannot change the description here without changing it in the code first. Now our test fails with the message: 
```
AssertionError: Mismatch in description for option --from-path=<path>.
07:40:12   E   == Docs ==
07:40:12   E   /path/to/directory-containing-dump-or-backup Path to a directory containing a database dump or backup file to upload.
07:40:12   E   == Command ==
07:40:12   E   /path/to/directory-containing-dump Path to a directory containing a database dump to upload.
```